Integrations/integration-VirusTotal-Private_API.yml

```diff
@@ -549,43 +556,101 @@ script:
         """
         corresponds to 'vt-get-url-report' command. Retrieves a report about a url
         """
-
         args = demisto.args()
-        url = args.get('resource')
+        urls = argToList(args.get('resource'))
         all_info = 1 if args.get('allInfo', None) == 'true' else 0
-        threshold = int(args.get('threshold', None) or demisto.params().get('urlThreshold', None) or 10);
         short_format = args.get('shortFormat', None) == 'true'
+        retries = int(args.get('retries', 2))
         full_response = args.get('fullResponse', None) == 'true'
+        threshold = int(args.get('threshold', None) or demisto.params().get('urlThreshold', None) or 10);
         if (full_response):
             max_len = 1000
         else:
             max_len = 50
 
-        response = get_url_report(url, all_info)
+        responses_dict = get_url_reports_with_retries(urls, all_info, retries)
+        md = ''
+        ec = {
+                'DBotScore': [],
+                outputPaths['url']: [],
+            }
+        for url, res in responses_dict.iteritems():
+            url_md, url_ec = create_url_report_output(url, res, ec, threshold, max_len, short_format)
+            md += url_md
+        if not md:
+            md = "No scans were completed in the elapsed time. Please run the command again in a few seconds."
 
-        if (response.get('response_code', None) == -1):
-            return "Invalid url"
+        completed_responses = responses_dict.values()
+        if len(completed_responses) == 1:
+            # This is done for backward compatibility
+            completed_responses = completed_responses[0]
 
-        positives = demisto.get(response, 'positives')
+        return {
+            'Type': entryTypes['note'],
+            'Contents': completed_responses,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': md,
+            'EntryContext': ec
+        }
 
-        md = '## VirusTotal URL report for: ' + url + '\n'
+
+    def get_url_reports_with_retries(urls, all_info, retries_left):
+        """
+        Returns dict of responses, where its keys are the URL related to the response.
+        """
+        requests_responses_dict = {}
+
+        for url in urls:
+            response = get_url_report(url, all_info)
+            if (response.get('response_code', None) == -1):
+                return_error("Invalid url provided: {}.".format(url))
+
+            if is_url_response_complete(response):
+                requests_responses_dict[url] = response
+
+        urls_scanned_count = len(requests_responses_dict)
+        urls_count = len(urls)
+        while urls_count != urls_scanned_count and retries_left:
```

Comment:
please change to a safer condition:
`while urls_count > urls_scanned_count and retries_left > 0`